nemo/collections/asr/models/ctc_bpe_models.py

```diff
@@ -90,6 +90,17 @@ def __init__(self, cfg: DictConfig, trainer=None):
         )
 
     def _setup_dataloader_from_config(self, config: Optional[Dict]):
+        if config.get("use_lhotse"):
+            from nemo.collections.asr.data.audio_to_text_lhotse import LhotseSpeechToTextBpeDataset
```

Comment:
Put these imports at the top of the file. If they are guarded properly there's no issue in importing the file.